include/itkExhaustiveLog.h

```diff
@@ -20,36 +20,36 @@
 
 #include "itkCommand.h"
 #include "itkExhaustiveOptimizerv4.h"
-/*
-namespace itk
-{
-    <template ImageType>
-    class ExhaustiveLog<ImageType> : public itk::Command
-    {
-    public:
-        using Self = ExhaustiveLog;
-        using Superclass = itk::Command;
-        using Pointer = itk::SmartPointer<Self>;
-        itkNewMacro(Self);
+ /*
```

Comment:
1-space indent does not seem right.

Also, why is all the code commented out?

Reply:
Code is commented in an attempt to isolate the root cause of the linking error in [Issue 2](https://github.com/InsightSoftwareConsortium/ITKOptimizationMonitor/issues/2).

It looks like VS is set up to format comments with 1-space indents. As I uncomment code I will take care to reformat accordingly.